lib/App/DuckPAN.pm

```diff
@@ -300,12 +300,12 @@ sub execute {
 				elsif($m eq 'ddg' && $ddg){ next }
 				push @modules, $_;
 			}
-			elsif (m/^duckpan|update|(upgrade|reinstall|latest)$/i) {
+			elsif (m/^duckpan|update|(upgrade|(reinstall|latest))$/i) {
```

Comment:
@zachthompson I've seen some CLI's that support both. `upgrade` could be used to upgrade installed packages and `update` would only update the DuckPAN client.

OSX's `homebrew` handles it the same way, whereas `npm` has an `update <pkg>` command.